src/js/modules/infragistics.ui.editors.js

```diff
@@ -9198,6 +9199,14 @@
 				}
 			}
 		},
+		_parseValueByIMEMask: function() {

```

Comment:
Actually, I think this should even be part of `_parseDateFromMaskedValue`, just like IME number conversion is part of `_parseNumericValueByMode` :)

Reply:
Or part of the respective editors' `_insert`